torch/_inductor/codegen/cpp.py

```diff
@@ -1266,19 +1266,19 @@ def load(self, name: str, index: sympy.Expr):
         if is_broadcast:
             if is_mask:
                 loadbuf = f"flag_to_float_scalar({loadbuf})"
-            line = f"at::vec::Vectorized<float>(static_cast<float>({loadbuf}))"
+            line = f"at::vec::Vectorized<float>(static_cast<float>({loadbuf}));"
```

Comment:
I don't think `;` is needed for load here since `cse.generate` would append the suffix (`;` in the case of cpp codegen) properly.

Reply:
Yes, changed.